kratos/tests/cpp_tests/sparse_matrices/run_my_test.py

```diff
@@ -0,0 +1,9 @@
+from KratosMultiphysics import *
```

Comment:
Seems to be a ignored file

Reply:
yes, i will clean this up before merging

Reply:
I only used this script